rhobs/alerting/data_plane/prometheus.release_service_alerts.yaml

```diff
@@ -0,0 +1,77 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: rhtap-release-service-alerting
+  labels:
+    tenant: rhtap
+spec:
+  groups:
+  - name: release_service_alerts
+    interval: 1m
+
+    rules:
+    # record rules for `ReleaseServiceProcessingDuration`
+    - record: rate_of_release_processing_duration_seconds_longer_than_1h_per_10m
+      expr: sum by (job) (rate(release_processing_duration_seconds_bucket{le="3600"}[10m]))
```

Comment:
The way it is right now, we're summing across clusters. Is that intentional? Should we consider the health of the service per-cluster or for all clusters together?

Reply:
We should also have that in mind on the alert rules summary/description. If it should be per-cluster, then we need to tell there which cluster the alert is for.

Reply:
Do we want the `release_processing_duration_seconds_bucket{le="3600"}` here or its compliment?
i.e. inf minus 1h.

Reply:
> The way it is right now, we're summing across clusters. Is that intentional? Should we consider the health of the service per-cluster or for all clusters together?

Not intentional. I just have no idea how these metrics are aggregated. In my mind I was adding alerts for "prod" regardless on where it runs.

> Do we want the `release_processing_duration_seconds_bucket{le="3600"}` here or its compliment? i.e. inf minus 1h.
The understanding I had from the prometheus doc was that if a release takes >=3600 secs it will be added here and `+Inf`. So AFAIU I would just need to take in consideration the data within the bucket that interests me. Is that correct?

Reply:
> Not intentional. I just have no idea how these metrics are aggregated. In my mind I was adding alerts for "prod" regardless on where it runs.

I suggest adding a test case that includes time series from multiple clusters and verify that things are aggregated correctly. Search for `source_cluster` in some of the other test files. You should be able to use something similar.

Reply:
I agree. I will add to the cases.

Reply:
@yftacherzog I added the source_cluster to the existent cases instead of adding more tests. Is that ok for you?

Reply:
Are you ok with the current behavior? I suspect that currently you're:
1. summing across clusters, which means it will only trigger when the aggregation across all cluster will apply to the threshold.
2. you're summary/description does not include the cluster label so even if you did not aggregate across multiple clusters, you'd have hard time knowing which cluster the alert is about.

Reply:
Correct me again if I am wrong @yftacherzog: 

I understand that SLO is not bounded to cluster but the service overall, so I think the cluster from where it comes (for SLO only) is not relevant. We might (and will soon) have more specific alerts where this information will be  relevant so we can take a direct action to it.

I added the `source_cluster` only to justify the `sum` in the query. Not a relevant information at this moment as said above. WDYT?

Reply:
I'm just highlighting what I see in the implementation. I leave it for you to decide as you have the understanding of the SLO and the requirements.

Reply:
Can you rebase so that we don't wait on that check that doesn't exist anymore?

Reply:
done